Mathlib/Algebra/Quandle.lean

```diff
@@ -152,7 +152,7 @@ identity holds.
 -/
 lemma act_act_self_eq (x y : S) : (x ◃ y) ◃ x = x ◃ y := by
   have h : (x ◃ y) ◃ x = (x ◃ y) ◃ (x ◃ 1) := by rw [act_one]
-  rw [h, ← Shelf.self_distrib, act_one]
+  rw [h, ←Shelf.self_distrib, act_one]
```

Comment:
@grunweg Before I rebased this PR on #15051, reviewdog complained about the lack of a space here, so maybe one of the linters was lost in the transition?

Reply:
Let me investigate: if I remove this space on master and run `lake exe lint-style` or `lake exe lint-style --fix` locally, I rightfully get an error: `Mathlib/Algebra/Quandle.lean : line 155 : ERR_ARR : Missing space after '←'.`
Thanks looks alright to me. Sorry.

Reply:
Looks like the error popped up in the run on the latest commit here too, but reviewdog didn't pick it up: https://github.com/leanprover-community/mathlib4/actions/runs/10415147022/job/28845191206?pr=15781#step:6:38

Since reviewdog just looks at git diff, maybe the fix option isn't writing the change to disk? (Haven't checked locally, sorry!)

Reply:
Let me check - it should do so (as the writing happens in lint-style.py, which is called just the same).